Makefile

```diff
@@ -77,7 +77,7 @@ TESTFLAGS ?= -short
 TESTFLAGSEE ?= -short
 
 # Packages lists
-TE_PACKAGES=$(shell $(GO) list ./...)
+TE_PACKAGES=$(shell $(GO) list ./...|grep -v plugin_tests)
```

Comment:
Why is this needed?

Reply:
done to ignore the folder with plugin tests, since they are ran by our test code, not by `go test`

Reply:
`go test` will ignore the anyway, as they are no go test files.

Reply:
This is still a concern.